crypto/bio/bss_dgram.c

```diff
@@ -1274,92 +1288,80 @@ static ossl_ssize_t dgram_sendmmsg(BIO *b, BIO_MSG *msg, size_t stride,
     if (num_msg == 0)
         return 0;
 
+    if (num_msg > OSSL_SSIZE_MAX)
+        num_msg = OSSL_SSIZE_MAX;
```

Comment:
Now this feels like duplicate with the BIO_MAX_MSGS_PER_CALL below. Should it be just replaced with that check? It won't be wrong even for the sendmsg/sendto implementations.

Reply:
The `BIO_MAX_MSGS_PER_CALL` test is specific to sendmmsg/recvmmsg. `BIO_MAX_MSGS_PER_CALL` is also only defined when using recvmmsg.

It may feel like a duplication, but my feeling here is keeping this separated out by method keeps the code more maintainable.

Reply:
OK